statsmodels/discrete/discrete_model.py

```diff
@@ -2704,6 +2706,345 @@ def fit_regularized(self, start_params=None, method='l1',
 
         return L1NegativeBinomialResultsWrapper(discretefit)
 
+class NegativeBinomialP(CountModel):
+    __doc__ = """
+    Negative Binomial model for count data
```

Comment:
better to distinguish from NegativeBinomial: 
"Generalized Negative Binomial (NB-P) model for count data"